src/core/CCList.mli

```diff
@@ -501,6 +501,11 @@ val sorted_merge : cmp:('a -> 'a -> int) -> 'a list -> 'a list -> 'a list
 (** [sorted_merge ~cmp l1 l2] merges elements from both sorted list using
     the given comparison function [cmp]. *)
 
+val sorted_diff : cmp:('a -> 'a -> int) -> 'a list -> 'a list -> 'a list
+(** [sorted_merge ~cmp l1 l2] returns the elements in [l1] that are not in [l2].
+    Both lists are assumed to be sorted with respect to [cmp] and
+    duplicate elements are treated individually. *)
```

Comment:
```suggestion
val sorted_diff : cmp:('a -> 'a -> int) -> 'a list -> 'a list -> 'a list
(** [sorted_diff ~cmp l1 l2] returns the elements in [l1] that are not in [l2].
    Both lists are assumed to be sorted with respect to [cmp] and
    duplicate elements are treated individually.
    @since NEXT_RELEASE *)
```